exonum/src/storage/proof_map_index/proof.rs

```diff
@@ -323,57 +321,55 @@ impl<V: fmt::Debug + StorageValue> MapProof<V> {
 }
 
 impl<V: fmt::Debug> BranchProofNode<V> {
-    fn validate(&self, searched_slice: &DBKey) -> Result<Option<&V>, Error> {
+    fn validate(&self, searched_path: &ProofPath) -> Result<Option<&V>, Error> {
         use self::BranchProofNode::*;
 
         // if we inspect the topmost level of a proof
         let res: Option<&V> = match *self {
             LeftBranch {
                 left_node: ref proof,
-                left_key: ref left_slice_key,
+                left_key: ref left_path,
```

Comment:
I wonder whether these renamings really make sense, given that fields in `BranchProofNode` variants are still named `*key`.